chainio/clients/fireblocks/contract_call.go

```diff
@@ -96,12 +96,19 @@ func NewContractCallRequest(
 			Calldata: calldata,
 		},
 		ReplaceTxByHash: replaceTxByHash,
-		GasPrice:        gasPrice,
 		GasLimit:        gasLimit,
-		MaxFee:          maxFee,
-		PriorityFee:     priorityFee,
-		FeeLevel:        feeLevel,
 	}
+
+	if maxFee != "" && priorityFee != "" {
+		req.MaxFee = maxFee
+		req.PriorityFee = priorityFee
+	} else if gasPrice != "" {
+		req.GasPrice = gasPrice
+	} else {
+		req.FeeLevel = feeLevel
```

Comment:
can `feeLevel` passed in from NewContractCallRequest be empty?

Reply:
Yes, in which case Fireblocks will default it to MEDIUM

Reply:
Might be helpful to leave a comment here about it's handled this way

Reply:
it's specified on line 57